pkg/controller/jenkins/configuration/base/reconcile.go

```diff
@@ -1092,3 +1133,39 @@ func (r *ReconcileJenkinsBaseConfiguration) handleAdmissionControllerChanges(cur
 		}
 	}
 }
+
+func verifyAPI(group string, version string) (bool, error) {
```

Comment:
we should move this to another file or package.
and as it returns a boolean, rename it: isXYZ or hasXYZ

Reply:
renamed it for now.